# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qscanq/runit.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/conf-svc.orig	2005-12-01 13:09:29.000000000 -0300
+++ ./src/conf-svc	2005-12-01 13:35:26.000000000 -0300
@@ -1,6 +1 @@
-/command/svc
-
-The first line of this file gives the full path to the svc program from
-Dan Bernstein's daemontools package. You shouldn't need to change it,
-unless you've installed daemontools to a non-standard place, or are using
-an outdated version of daemontools.
+/command/runsvctrl
--- ./src/run-cleanq.c.orig	2005-12-01 13:09:29.000000000 -0300
+++ ./src/run-cleanq.c	2005-12-01 13:31:55.000000000 -0300
@@ -7,7 +7,7 @@
 
 char *svco_cmd[] = {
 (char *)auto_svc,
-"-o",
+"o",
 (char *)auto_scanq,
 0
 };
--- ./src/install-cleanq.c.orig	2005-12-02 19:32:22.000000000 -0300
+++ ./src/install-cleanq.c	2005-12-02 19:51:45.000000000 -0300
@@ -79,6 +79,7 @@
   if (chdir("log") == -1)
     strerr_die4sys(111,FATAL,"unable to chdir to ", r, "/log: ");
   makedir("main",02755,auto_loguid,0);
+  fn = "main/config"; start(); outs(auto_cqlog); finish();
 
   /* Create the "run" script */
   fn = "run";
@@ -87,9 +88,7 @@
     start();
     outs("#!/bin/sh\nexec setuidgid ");
     outs(auto_loguser);
-    outs(" multilog t \\\n  ");
-    outs(auto_cqlog);
-    outs("\\\n  ./main\n");
+    outs(" svlogd -t ./main\n");
     finish();
     move_file(fn,"run");
     fn = "run";
